deps.edn

```diff
@@ -0,0 +1,8 @@
+{:deps
+ {nrepl                 {:mvn/version "0.6.0"}
+  org.clojure/clojure   {:mvn/version "1.10.0"}
```

Comment:
Can we do without this?

Reply:
### TL;DR

I'd like to keep it.

### Thoughts

I _like_ when the clojure version is explicitly specified instead of implicit depending on environment or other libraries bringing it in. But just because I like it doesn't mean it's a good idea. However, I couldn't find a common pattern online or reasonings not to include it.

One benefit of having it, is that it signals when the library was created and what it was tested to work with.

I looked online to try to find if people who are using `tools.deps` are using something like `:scope "provided"` as they are with `leiningen`, but it doesn't seem like they are. And given one comment online, `tools.deps` `:aliases` are preferred over `:scope`, as `:scope` is a maven specific thing and aliases can be applied and used to override anything.

For example, one can setup local aliases in `~/.clojure/deps.edn` like this one: https://github.com/seancorfield/dot-clojure/blob/master/deps.edn
And add whatever clojure they like at runtime.

Projects using this library should be able to override this clojure version also.